lib/tasks/cssbundling/build.rake

```diff
@@ -2,17 +2,21 @@ namespace :css do
   desc "Build your CSS bundle"
   task :build do
     unless system "yarn install && yarn build:css"
-      raise "cssbundling-rails: Command css:build failed, ensure yarn is installed and `yarn build:css` runs without errors"
+      raise "cssbundling-rails: Command css:build failed, ensure yarn is installed and `yarn build:css` runs without errors or use CSS_BUILD=false env variable"
     end
   end
 end
 
-if Rake::Task.task_defined?("assets:precompile")
-  Rake::Task["assets:precompile"].enhance(["css:build"])
-end
+skip_css_build = %w(no false n f).include?(ENV["CSS_BUILD"])
```

Comment:
Restate the ENV as `SKIP_CSS_BUILD` and you can just check for the existence of that ENV rather than try to predict its value.

Reply:
done